### ISSUE TITLE
Add Windows setup information in Chinese

### ISSUE THREAD

**Issue by jenweber** [cardstack/cardstack-documentation#31]
Note: This issue already has a volunteer

It would be great to have some learning resources in Chinese! Here are the suggested steps to add a new article to these docs:

- Create a folder called `chinese` within `guides/release`
- in that new directory make a file called `windows-setup.md` or similar, and a file called `index.md`
- Edit `pages.yml` to include the section and pages (example below)
- Add content to windows-setup
- In index.md, add one or two sentences that are a welcome message. An index.md file is required for the new section to render.
- Add any images you need in `public/images`

Example pages.yml
```
- title: "Chinese Learning Resources"
  url: "chinese"
  pages:
    - title: "Page title goes here"
      url: "index"
    - title: "Page title goes here"
      url: "windows-setup"
```

Thank you for the help!

**Comment by hexixiaozilong** [comment]
Recommended pages.yaml:

- title: "Chinese Learning Resources"
  url: "chinese-learning"
  pages:
    - title: "Windows安装教程"
      url: "windows-environment-setup-cn"

Some comments to the documentation itself:
请在文档最开始的地方说明：这个教程是在Windows操作系统上安装Cardstack开源全栈应用[Cardboard](https://github.com/cardstack/cardboard)的流程。同样的安装流程可以应用到其他的开源全站应用上，例如[Cardfolio](https://github.com/cardstack/portfolio)。

**Comment by jenweber** [comment]
Closed with #32
